angularjs-portal-home/src/main/webapp/my-app/search/partials/search-results.html

```diff
@@ -26,29 +26,21 @@ <h4 class="header">MyUW</h4>
       No MyUW results. <a href="apps">Try browsing instead?</a>
     </div>
     <div ng-repeat="portlet in myuwFilteredResults = (myuwResults | filter:searchTermFilter | showApplicable:showAll | orderBy:sortParameter | limitTo:searchResultLimit)" class="result">
-      <h4><a ng-href="{{getLaunchURL(portlet)}}" target="{{::portlet.target}}">{{ portlet.name }}</a></h4>
+      <h4><a ng-href="{{getLaunchURL(portlet)}}" target="{{::portlet.target}}">{{ portlet.name }} <i ng-if='GuestMode && !portlet.canAdd' class="fa fa-external-link"></i></a> <small ng-if='GuestMode && !portlet.canAdd'>(login to use)</small></h4>
```

Comment:
I'd skip the `<i class="fa fa-external-link"></i>`, considering the "(login to use)" text enough.